storybook/qmlTests/tests/tst_TokenSelectorView.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.15
+import QtTest 1.15
+
+import Models 1.0
+
+import AppLayouts.Wallet.views 1.0
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.stores 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    QtObject {
+        id: d
+
+        readonly property var flatNetworks: NetworksModel.flatNetworks
+        readonly property var currencyStore: CurrenciesStore {}
+        readonly property var assetsStore: WalletAssetsStore {
+            id: thisWalletAssetStore
+            walletTokensStore: TokensStore {
+                plainTokensBySymbolModel: TokensBySymbolModel {}
+            }
+            readonly property var baseGroupedAccountAssetModel: GroupedAccountsAssetsModel {}
+            assetsWithFilteredBalances: thisWalletAssetStore.groupedAccountsAssetsModel
+        }
+
+        readonly property var adaptor: TokenSelectorViewAdaptor {
+            assetsModel: d.assetsStore.groupedAccountAssetsModel
+            flatNetworksModel: d.flatNetworks
+            currentCurrency: d.currencyStore.currentCurrency
+        }
+    }
+
+    Component {
+        id: componentUnderTest
+        TokenSelectorView {
+            anchors.fill: parent
+
+            model: d.adaptor.outputAssetsModel
+        }
+    }
```

Comment:
What is needed here is just simple list model done according to the specification in the component's API: https://github.com/status-im/status-desktop/pull/15157/files#diff-3de6ffcec9aefd08e1e130e2044ed30fb4d7502ce958ae568faa168ff07d694cR9
```
 Component {
        id: componentUnderTest
        TokenSelectorView {
            anchors.fill: parent

            model: ListModel {
                   ListElement { <some content here> } 
            }
        }
    }
```